node_modules/oae-authentication/emailTemplates/reset.shared.js

```diff
@@ -0,0 +1,113 @@
+ * passed in, it will be prefixed with the base url.
+ *
+ * @param  {String}     link        The link to check
+ * @param  {String}     baseUrl     The base url that can be used to prefix relative urls
+ * @return {String}                 The absolute link prefixed with the base url
+ */
+var ensureAbsoluteLink = module.exports.ensureAbsoluteLink = function(link, baseUrl) {
+    // If the link is empty or null, we return the empty string. This can happen when
+    // we try to link a private user (private users are scrubbed and have no profile path)
+    if (!link) {
+        return '';
+
+    // If the link already has `http` in it (e.g., twitter profile pics) we return as-is
+    } else if (link.indexOf('http') === 0) {
+        return link;
+
+    // Otherwise we prefix it with the base url
+    } else {
+        return baseUrl + link;
+    }
+};
+
+/**
+ * Ensure that each link in an HTML fragment is an abolute url, If a relative link is
+ * found, it will be prefixed with the base url.
+ *
+ * @param  {String}     str         The html string in which to check for absolute links
+ * @param  {String}     baseUrl     The base url that can be used to prefix relative urls
+ * @return {String}                 The html in which each link is absolute
+ */
+var ensureAbsoluteLinks = module.exports.ensureAbsoluteLinks = function(str, baseUrl) {
+    var html = $('<div>' + str + '</div>');
+    html.find('a').each(function(i, elem) {
+        var link = $(this).attr('href');
+        link = ensureAbsoluteLink(link, baseUrl);
+        $(this).attr('href', link);
+    });
+    return html.html();
+};
+
+/**
+ * Get an appropariate summary given a user's email preference and a set of activities
+ * The options are:
+ *     -   When an immediate email is sent with a single actor and a single activity: `<displayName> has been active recently`
+ *     -   When an immediate email is sent with a single actor and multiple activities: `<displayName> has been very active recently`
+ *     -   When an immediate email is sent with multiple actors: `New activity is waiting for you`
+ *     -   When a daily email is sent: `Here's your summary of today's activity`
+ *     -   When a weekly email is sent: `Here's your summary of last week's activity`
+ *
+ * @param  {Object}         util                The template utility that can be used to encode HTML, translate keys, etc
+ * @param  {String}         emailPreference     The email preference for which to generate an email summary
+ * @param  {Activity[]}     activitities        The activities for which to generate a summary
+ * @param  {String}         baseUrl             The base url that each link should have as a prefix
+ * @return {String}                             An appropriate summary given the user's email preference and given activities
+ */
+var getEmailSummary = module.exports.getEmailSummary = function(util, emailPreference, activities, baseUrl) {
```

Comment:
It doesn't look like you're using this function. This can be removed. As for the `ensureAbsoluteLink*` functions, we need to find a way to consolidate them with the oae-activity/email.shared logic, but that can be outscoped for now